pkg/pgmodel/querier/query_builder_samples.go

```diff
@@ -213,6 +213,10 @@ func buildSingleMetricSamplesQuery(metadata *evalMetadata) (string, []interface{
 		start, end = metadata.timeFilter.start, metadata.timeFilter.end
 	}
 
+	if filter.useDownsamplingViews {
```

Comment:
See code around https://github.com/timescale/promscale_extension/pull/596/files#r1060609175 for more info.